bigtable/doc/bigtable-hello-world.md

```diff
@@ -0,0 +1,90 @@
+# Example: C++ Hello World Application
+
+This example is a very simple "hello world" application, written in C++, that
+illustrates how to:
+
+* Connect to a Cloud Bigtable instance.
+* Create a new table.
+* Write data to the table.
+* Read the data back.
+* Delete the table.
+
+## Running the example
+
+This example uses the
+[Cloud Bigtable C++ Client Library](https://GoogleCloudPlatform.github.io/) to
+communicate with Cloud Bigtable.
+
+To run the example program, follow the
+[instructions](https://github.com/GoogleCloudPlatform/google-cloud-cpp/tree/master/bigtable/examples/)
+for the example on GitHub.
+
+### Including the Necessary Headers
+
+The example uses the following headers:
+
+@snippet bigtable_hello_world.cc bigtable includes
+
+### Connecting to Cloud Bigtable to manage tables
+
+To manage tables, connect to Cloud Bigtable using
+`bigtable::CreateDefaultAdminClient()`:
+
+@snippet bigtable_hello_world.cc connect admin
+
+### Creating a table
+
+Create a table with `bigtable::TableAdmin::CreateTable()`:
+
+@snippet bigtable_hello_world.cc create table
+
+@see `bigtable::v0::TableAdmin` for additional operations to list, read, modify,
```

Comment:
This also doesn't render anything special.

Reply:
Ditto, Doxygen tag.